.github/workflows/e2e.yml

```diff
@@ -0,0 +1,40 @@
+name: E2E Test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test:
+    name: e2e
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Create workspace
+        id: workspace
+        uses:  TakeScoop/terraform-cloud-workspace-action@v2
+        with:
+          terraform_organization: takescoop-oss
+          terraform_token: ${{ secrets.TF_TOKEN_OSS }}
+          name: remote-state-action-e2e-test
+          apply: true
+          working_directory: test-e2e/
+          vcs_type: github
+
+      - name: Run action
+        id: action
+        uses: ./
+        with:
+          workspace: remote-state-action-e2e-test
+          organization: takescoop-oss
+          token: ${{ secrets.TF_TOKEN_OSS }}
+      
+      - name: Test simple output
```

Comment:
Cases to fix/add:

* Multiline - don't mix this up with JSON
* JSON - via `jsonencode`, without whitespace, so it purely tests JSON handling
* Numerics, int and float
* Complex types, e.g. tuples, arrays, sets, maps - these are likely to bugs that require additional logic to resolve, e.g. JSON encoding.